HUD/AddBlipForEntity.md

```diff
@@ -7,7 +7,7 @@ ns: HUD
 // 0x5CDE92C702A8FCE7 0x30822554
 Blip ADD_BLIP_FOR_ENTITY(Entity entity);
 ```
-Create a blip that for default is red (enemy), you can use the [SET_BLIP_AS_FRIENDLY](#_0xC6F43D0E) to make it blue (friend)
+Create a blip that for default is red (enemy), you can use [SET_BLIP_AS_FRIENDLY](#_0xC6F43D0E) to make it blue (friend)
```

Comment:
for default -> by default

not sure if this applies to non-peds either.

Reply:
Tested and I could make objects, as well as vehicles 'enemies'.